helpers.go

```diff
@@ -33,10 +39,20 @@ func fromLibdns(zone string, records []libdns.Record) ([]leasewebRecordSet, erro
 		}
 		currentRecordInfo.consumed = true
 
+		// Cleanup record name and ensure it ends with domain.ext[dot] even if dns_challenge_override_domain is set
+		// trimming both zone & domainName is probably overzealous, but better be safe then sorry
+		var recordName = fmt.Sprintf("%s.%s", strings.TrimSuffix(strings.TrimSuffix(currentRecordInfo.libdnsRecord.Name, zone), domainName), zone)
+		var recordTTL = int(currentRecordInfo.libdnsRecord.TTL.Seconds())
+		if !slices.Contains(supportedTTLs, recordTTL) {
+			// Use the first listed TTL if the user did not provide a TTL or provided a unsupported value
+			// It would probably be nice to log a warning about this, but that doesn't seem to be supported in libdns
+			recordTTL = supportedTTLs[0]
```

Comment:
I agree with the approach, but maybe we could default to 3600 (1 hour) as that is what the web UI of Leaseweb also defaults to.

Reply:
Hey, you're right. I was just thinking pure DNS but it makes sense to keep it low of course!